src/core/util/sys_vars.cpp

```diff
@@ -1034,83 +1034,34 @@ void mce_sys_var::get_env_params()
 
 #ifdef DEFINED_NGINX
     case MCE_SPEC_NGINX:
```

Comment:
put `// Fallthrough` comment to silent possible compiler warnings

Reply:
Done